pom.xml

```diff
@@ -730,6 +730,15 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>jdk17</id>
+			<activation>
+				<jdk>[17,)</jdk>
+			</activation>
+			<properties>
+				<test.additional.args> --add-opens java.base/java.lang=ALL-UNNAMED </test.additional.args>
```

Comment:
What does this line mean?

Reply:
This line means opens `java.lang` module in java.base

`apollo` unit test code is development under java8，adding this line to avoid `module java.base does not "opens java.lang" to unnamed module @224aed64` error.